ledger_device_sdk/src/nbgl.rs

```diff
@@ -179,37 +135,17 @@ const INFO_FIELDS: [*const c_char; 2] = [
 pub struct NbglHomeAndSettings<'a> {
     glyph: Option<&'a NbglGlyph<'a>>,
     // app_name, version, author
-    info_contents: [*const c_char; 3],
-    settings_contents: nbgl_content_t,
+    info_contents: Vec<CString>,
```

Comment:
I don't think we need a Vec here as we only need 3 strings : version, developer, app_name.

Reply:
Nowadays yes, but more future-proof if we use a `Vec`